### PR TITLE
docs: Create ace-demo.html

### PR DESCRIPTION
**Pull request by vanillajonathan**
This is a demo of using [Ace](https://ace.c9.io/) with the PRQL syntax highlighting mode.

### CODE REVIEW

**Comment by max-sixty** [comment]
OK cool. 

Would be nice to have some sort of test / check that it's still working correctly, but I guess that's difficult with HTML.

Shall we link to this from the grammars readme? Otherwise it's totally orphaned.

Separately, we could add the link from https://github.com/PRQL/prql/pull/3493#issuecomment-1721627217 too.

**Comment by max-sixty** [comment]
If you want to spend more time with grammars, adding tests for our existing grammars would be really helpful.

And one *huge* win would be allowing the PRQL lezer grammar to work in Jupyter notebooks. The grammar is our best grammar, and is the format (CodeMirror) required for Jupyter notebooks, but the integration hasn't been writen. That would be a difficult project but would immensely valuable.

File: grammars/README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ an index.
 
 - [Ace](https://ace.c9.io/) — supported. The grammar is upstream
   ([prql_highlight_rules.js](https://github.com/ajaxorg/ace/blob/master/src/mode/prql_highlight_rules.js)).
+  See the
+  [demo](https://htmlpreview.github.io/?https://github.com/PRQL/prql/blob/main/grammars/ace-demo.html).
```
Comment (max-sixty) [inline grammars/README.md]: Does this work? I'm not sure I'm doing it right but if I try https://htmlpreview.github.io/?https://github.com/PRQL/prql/blob/0bcb3d530aa9eb411a9c09380738732f5be1407f//grammars/ace-demo.html, I just get some text.

Comment (vanillajonathan) [reply grammars/README.md]: Ugh, I had hoped so, but I guess not. I can move to `/web/website/content/` though.

Comment (max-sixty) [reply grammars/README.md]: OK, let's try that

Comment (max-sixty) [reply grammars/README.md]: (need to update link too :) )

Comment (vanillajonathan) [reply grammars/README.md]: I hope this works!

File: grammars/README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ an index.
 
 - [Ace](https://ace.c9.io/) — supported. The grammar is upstream
   ([prql_highlight_rules.js](https://github.com/ajaxorg/ace/blob/master/src/mode/prql_highlight_rules.js)).
+  See the [demo](https://prql-lang.org/ace-demo).
```
Comment (max-sixty) [inline grammars/README.md]: I didn't realize this was a top-level page. Could we put it down a level to indicate that it's not a sibling of the key pages? Maybe a `content/demos/ace-demo.html` page?

Comment (vanillajonathan) [reply grammars/README.md]: Sounds good.

**Comment by max-sixty** [comment]
We're doing a release v soon anyway, so it'll publish then

**Pull request merged**